services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/dataaccess/RestrictionInformationManagerImpl.java

```diff
@@ -34,15 +36,17 @@ public RestrictionInformationResponse getRestrictionInformation(UserInfo userInf
 			throw new IllegalArgumentException("Unsupported type: " + request.getRestrictableObjectType());
 		}
 
```

Comment:
will have two methods in the util, one for entity and one for team., this method needs to use both utils.